### PR TITLE
958 testimonial

### PR DESCRIPTION
**Pull request by hippogriffic**

### CODE REVIEW

**Comment by hippogriffic** [comment]
also contains spacing for #974

File: django-verdant/rca/static/rca/css/components/slider.less [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        width: 27px;
+        height: 49px;
+        z-index: 1;
+
+        &--next {
+            right: 16px;
+            top: calc(~"50% - 31px");
+
+            @media @media-desktop-regular {
+                top: 30%;
```
Comment (helenb) [inline django-verdant/rca/static/rca/css/components/slider.less]: The positioning of the arrows will vary according the block, and the events one has a different size at desktop to mobile - so not all of this is totally reusable. 

Also the previous button being hidden is going to be ditched as that was the old mobile design for the showcase - the new one shows both buttons.

File: django-verdant/rca/static/rca/css/streamfield-blocks/showcase.less [hunk 2]
```diff
@@ -122,124 +122,125 @@
 
 // SLICK STYLE OVERRIDES
 // Much of this is layout rejigging to account for the large central slide
+.showcase {
```
Comment (helenb) [inline django-verdant/rca/static/rca/css/streamfield-blocks/showcase.less]: I think we both made the same change here in separate branches, in order to scope the slick styles. The only difference was that I didn't repeat the `.showcase` - yours is perhaps slightly clearer.

File: django-verdant/rca/static/rca/js/programme-finder.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// Programme Finder
+$(function() {
+    $("input#course_search").autocomplete({
```
Comment (helenb) [inline django-verdant/rca/static/rca/js/programme-finder.js]: can we change this to a `js-` prefixed class selector?

File: django-verdant/rca/static/rca/css/components/homepage-block.less [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.homepage-block {
+    margin-bottom: @spacing-onepointfive;
```
Comment (helenb) [inline django-verdant/rca/static/rca/css/components/homepage-block.less]: We could potentially add a variant here for the blocks which aren't full-width - realised that currently all mine are, and I need to adapt that.

File: django-verdant/rca/templates/rca/blocks/testimonial_block.html [hunk 5]
```diff
@@ -1,9 +1,22 @@
...
-{% image value.image original %}
+<div class="testimonial__slide">
+    <div class="testimonial__content">
+        {% image value.image fill-600x335 class="testimonial__image" %}
 
-<p>{{ value.quote }}</p>
+        <blockquote class="testimonial__quote">
+            <p class="testimonial__quote-text">
+                <span class="testimonial__quote--highlight">
+                    <svg class="testimonial__quote-icon">
```
Comment (helenb) [inline django-verdant/rca/templates/rca/blocks/testimonial_block.html]: You can add an `aria-hidden="true"` here, as the quote mark is purely decorative.

File: django-verdant/rca/static/rca/js/showcase-block.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+$(function(){
+    // Showcase carousel
+    var $showcaseSlider = $('.js-showcase');
+        var prevButton = '<button aria-label="Previous" class="slider__button slider__button--prev" type="button"><svg class="slider__arrow slider__arrow--prev"><title>Previous</title><use xlink:href="#chevron"></use></svg></button>';
+        var nextButton = '<button aria-label="Next" class="slider__button slider__button--next" type="button"><svg class="slider__arrow slider__arrow--next"><title>Previous</title><use xlink:href="#chevron"></use></svg></button>';
+
+    $showcaseSlider.slick({
```
Comment (helenb) [inline django-verdant/rca/static/rca/js/showcase-block.js]: I think we should only initiate the carousel if there is more than one testimonial - seems an element where there is likely to be only 1?

**Pull request merged**